src/ResourceManagement/Compute/Domain/InterfaceImpl/VirtualMachineScaleSetImpl.cs

```diff
@@ -345,7 +345,7 @@ System.Collections.Generic.IReadOnlyList<string> Microsoft.Azure.Management.Comp
         /// <summary>
         /// Gets the eviction policy of the virtual machines in the scale set.
         /// </summary>
-        Models.VirtualMachineEvictionPolicyTypes Microsoft.Azure.Management.Compute.Fluent.IVirtualMachineScaleSetBeta.VirtualMachineEvictionPolicy
+        string Microsoft.Azure.Management.Compute.Fluent.IVirtualMachineScaleSetBeta.VirtualMachineEvictionPolicy
```

Comment:
Like VM Size enum it would be good to provide [`Models.VirtualMachineEvictionPolicyTypes`](https://github.com/Azure/azure-libraries-for-net/blob/master/src/ResourceManagement/Compute/Generated/Models/VirtualMachineEvictionPolicyTypes.cs) as enum for usability.

Reply:
Again, this is generated stuff : https://github.com/Azure/azure-libraries-for-net/pull/608/files#diff-faf6992f155acccd61884d14ee2e13c1L19

I didnt hand edit it :(